OpenRA.Mods.Common/Traits/Render/WithRotor.cs

```diff
@@ -28,9 +28,6 @@ public class WithRotorInfo : ITraitInfo, IRenderActorPreviewSpritesInfo, Require
 		[Desc("Position relative to body")]
 		public readonly WVec Offset = WVec.Zero;
 
-		[Desc("Change this when using this trait multiple times on the same actor.")]
-		public readonly string Id = "rotor";
```

Comment:
We don't need it as the game won't error if these stay in the rules, but I would like to see the now junk rules removed from https://github.com/pchote/OpenRA/blob/animation-keys/mods/ra/rules/aircraft.yaml#L213 etc.